tests/subsys/logging/log_core_additional/log2.conf

```diff
@@ -1,6 +1,6 @@
 CONFIG_MAIN_THREAD_PRIORITY=5
 CONFIG_ZTEST=y
-CONFIG_TEST_USERSPACE=y
+#CONFIG_TEST_USERSPACE=y
```

Comment:
Delete this or else someone might think it was accidentally commented out.

Reply:
Thank you for correcting this!  @SebastianBoe ~